### ISSUE TITLE
Missing file data/maastokuvionreuna.geojson

### ISSUE THREAD

**Issue by mathias-nyman** [vokkim/finnish-nautical-chart-vectors#15]
The commit https://github.com/vokkim/finnish-nautical-chart-vectors/commit/7ae66fe46b7cf222bf035d713ecb1449ef64a70e added processing of a `data/maastokuvionreuna.geojson` file but does not create or download it.

Error:
```
node import/prepare-geojson-data.sh
...
[i] Error: File not found (data/maastokuvionreuna.geojson)
```

**Comment by vokkim** [comment]
Yeah that's probably some data I tested with, but did not end up in the chart yet.
